tachyon/zk/expressions/evaluator/proving_evaluator.h

```diff
@@ -27,14 +27,14 @@
 namespace tachyon::zk {
 
 template <typename Evals>
-class SimpleEvaluator
+class ProvingEvaluator
```

Comment:
how about "LookupEvaluator"?
That's why, It is used for only Lookup compression.

Reply:
Good point